websites/N/Nutaku/dist/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "Cascade",
+    "id": "442612628104151040"
+  },
+  "service": "Nutaku",
+  "description": {
+    "en": "The best Hentai Games online on Nutaku. Your number 1 destination for uncensored browser, mobile and downloadable hentai games."
+  },
+  "regExp": "([a-z]+[.])nutaku.net[/]",
+  "url": ["nutaku.net", "www.nutaku.net"],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/fvSnqjK.png",
+  "thumbnail": "https://i.imgur.com/h9zq9na.png",
```

Comment:
could you change these images on screenshot to idk, our logo? 😂

Reply:
it is their logo, just picked their accent/primary color. its all in their branding

Reply:
If I think correctly from what ririxi said, ririxi asking you to change the images on thumbnail to something like this
![a](https://i.imgur.com/UrX3fAS.png)

Reply:
[image](https://i.imgur.com/ypxOFV9.jpg)

Reply:
good too 👍